Swagger/PlayFab/Authentication.swagger.json

```diff
@@ -96,15 +96,13 @@
     }
   },
   "definitions": {
-    "ApiErrorWrapper": {
+    "ApiError": {
       "description": "The basic wrapper around every failed API response",
       "type": "object",
       "properties": {
         "code": {
           "description": "Numerical HTTP code",
-          "type": "number",
-          "format": "int32",
```

Comment:
I'm hesitant to remove fields, even if both of us don't need them.
This might be a field that's required by MSDN.